charts/cluster-api-cluster-openstack/values.yaml

```diff
@@ -43,13 +43,6 @@ openstack:
 # Cluster wide configuration.
 #
 # cluster:
-  # Applies taints to all nodes on creation.  Once a node is provisioned you
```

Comment:
Why is this being removed?  It's documentation surely?

Reply:
It is, but the example is redundant now as Cilium automatically taints nodes. We could stick something else in, but I felt that was a confusing thing to have in there as adding that and installing the cilium chart with it's defaults will cause a conflict and prevent the second control planes coming online, unless the user specifically tells cilium not to add the the taint. 

Maybe we leave it in with a note about cilium adding the taint unless the user says not to?

Reply:
Updated in the other PR relating to charts and doc updates